test/test_run_analysis.rb

```diff
@@ -50,6 +50,88 @@ def _assert_and_puts(output, msg, expect_error = true)
+      next if _expected_warning_message(message, "Both 'dishwasher' schedule file and weekday fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'dishwasher' schedule file and weekend fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'dishwasher' schedule file and monthly multipliers provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'cooking_range' schedule file and weekday fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'cooking_range' schedule file and weekend fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'cooking_range' schedule file and monthly multipliers provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'hot_water_fixtures' schedule file and weekday fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'hot_water_fixtures' schedule file and weekend fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'hot_water_fixtures' schedule file and monthly multipliers provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'plug_loads_other' schedule file and weekday fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'plug_loads_other' schedule file and weekend fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'plug_loads_other' schedule file and monthly multipliers provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'lighting_garage' schedule file and weekday fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'lighting_garage' schedule file and weekend fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'lighting_garage' schedule file and monthly multipliers provided; the latter will be ignored.")
+      next if _expected_warning_message(message, 'Could not find state average propane rate based on')
+      next if _expected_warning_message(message, 'Could not find state average fuel oil rate based on')
+      next if _expected_warning_message(message, "Specified incompatible corridor; setting corridor position to 'Single Exterior (Front)'.")
+      next if _expected_warning_message(message, 'DistanceToTopOfWindow is greater than 12 feet; this may indicate incorrect units. [context: /HPXML/Building/BuildingDetails/Enclosure/Windows/Window/Overhangs[number(Depth) > 0]')
+      next if _expected_warning_message(message, 'Home with conditioned basement has floor insulation.')
```

Comment:
This should be checked too.

Reply:
When `Insulation Floor=Uninsulated` => `floor_over_foundation_assembly_r=5.3`.

Reply:
That seems like a high R-value to me. Who calculated it and how?

Reply:
@aspeake Does the above ring a bell for you?

Reply:
I'm not sure exactly how we came up with those numbers but I tracked this down: https://github.com/NREL/resstock/pull/792

For some reason we decided to go from 3.1 --> 5.3, at that point I think we were using numbers from v2.5 or BEopt? I see BEopt 2.8 uses 3.1, maybe that + floor covering?

Reply:
@shorowit @aspeake Should we leave this at 5.3?

Reply:
I guess it is assuming wood flooring + carpet, so that is fine. I suggest we bump up the value at which OS-HPXML triggers a warning then. Let's change it from 2.1 to 6.0.